src/src/tx_main.cpp

```diff
@@ -417,17 +417,37 @@ void ICACHE_RAM_ATTR SendRCdataToRF()
 
   // Sync spam only happens on slot 1 and 2 and can't be disabled
   if ((syncSpamCounter || WithinSyncSpamResidualWindow) && (NonceFHSSresult == 1 || NonceFHSSresult == 2))
-  {
-    otaPkt.std.type = PACKET_TYPE_SYNC;
-    GenerateSyncPacketData(OtaIsFullRes ? &otaPkt.full.sync.sync : &otaPkt.std.sync);
+  { 
+    if(OtaIsFullRes)
+    {
+      otaPkt.full.sync.packetType = PACKET_TYPE_SYNC;
+      otaPkt.full.sync.telemetryStatus = TelemetryReceiver.GetCurrentConfirm();
+      otaPkt.full.sync.zero = 0;
```

Comment:
Is this necessary?
Won't these remaining bits be ignored on the other end anyway?

Reply:
Yes, these bits are ignored. But, by defining the all unused bits in the headers as "zero" and always setting them to a fixed value, makes it easier to add future extensions.

For example, it would have been possible to implement this PR without breaking OTA compatibility if the current packet would have defined as:
```
        struct {
            uint8_t packetType: 2,
                    packageIndex: 4,
                    zero: 2;
            uint8_t payload[ELRS8_MSP_BYTES_PER_CALL];
        } msp_ul;
```
instead of:
```
        struct {
            uint8_t packetType: 2,
                    packageIndex: 6;
            uint8_t payload[ELRS8_MSP_BYTES_PER_CALL];
        } msp_ul;
```
Even though the possible values for packageIndex are 0-14, the field is 6 bits long. In other words, only the low 4 bits are used and the 2 high bits are always 0. 

However, as the current code accesses packageIndex as 6 bits, we can not use the high two bits for something else without breaking OTA compatibility...